sdk/trafficmanager/Azure.ResourceManager.TrafficManager/samples/Generated/Samples/Sample_SubscriptionResourceExtensions.cs

```diff
@@ -41,17 +41,17 @@ public async Task CheckTrafficManagerNameAvailabilityV2_NameAvailabilityV2TestNa
                 Name = "azsmnet5403",
                 ResourceType = new ResourceType("microsoft.network/trafficmanagerprofiles"),
             };
-            TrafficManagerNameAvailabilityResult result = await subscriptionResource.CheckTrafficManagerNameAvailabilityV2Async(content);
+            TrafficManagerNameAvailabilityResult result = await subscriptionResource.CheckTrafficManagerNameAvailabilityV2ProfileAsync(content);
```

Comment:
mapping failed: https://github.com/Azure/azure-sdk-for-net/blob/91af14e56d8b9f12d66fc5217e2e13670a21d8e1/sdk/trafficmanager/Azure.ResourceManager.TrafficManager/src/autorest.md?plain=1#L86